lib/CL/pocl_binary.c

```diff
@@ -52,9 +52,10 @@
                           no sense in binary, and whether argument is local
                           is already in cl_kernel_arg_address_qualifier);
                           add has_arg_metadata & kernel attributes */
+/* changes for version 8: added local_alignments after local_sizes */
 
 #define FIRST_SUPPORTED_POCLCC_VERSION 6
-#define POCLCC_VERSION 7
+#define POCLCC_VERSION 8
```

Comment:
It's not your define, but I think a more descriptive name here is POCL_BINARY_VERSION.